recipe_scrapers/barefootcontessa.py

```diff
@@ -0,0 +1,61 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class BareFootContessa(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "barefootcontessa.com"
+
+    def author(self):
+        return "barefootcontessa.com"
```

Comment:
I wonder whether we should return Ina Garten as the author name - what do you reckon?  (it looks like a website where all the recipes are written by them - I could be mistaken, though)

(there's a `humans.txt` file on the site too, and that's neat - it's credits for the page design authors and technology providers rather than content authors though)

Reply:
Absolutely should, it does appear to be exclusively Ina's recipes. This site contains a huge amount of great looking recipes gonna have to add some of these to my cooking list.